application/controllers/Widgets.php

```diff
@@ -81,16 +81,17 @@ public function qsos($logbook_slug = null) {
 
 	public function oqrs($user_callsign = 'CALL MISSING') {
 		$this->load->model('oqrs_model');
-		$stations = $this->oqrs_model->get_oqrs_stations();
+		$data['slug'] = $this->input->get('slug', TRUE);
+		$data['userid'] = $this->publicsearch->get_userid_for_slug($data['slug']);
```

Comment:
You need to load the public search model

```suggestion
		$this->load->model('oqrs_model');
		$this->load->model('publicsearch');
		$data['slug'] = $this->input->get('slug', TRUE);
		$data['userid'] = $this->publicsearch->get_userid_for_slug($data['slug']);
```